rdf_diff_store/models.py

```diff
@@ -0,0 +1,25 @@
+# generated by fastapi-codegen:
+#   filename:  openapi.yaml
+#   timestamp: 2022-09-07T12:21:13+00:00
+
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class Error(BaseModel):
```

Comment:
Litt ukomfortabel med klassenamn `Error`. Er litt for generelt.

Reply:
Renamet `HTTPError`, er det bedre?